### DISCUSSION TITLE
Can we update Tailwind version

### DISCUSSION THREAD

**Discussion by shoaibsharif** [beyondcode/devtoolsfortailwind-community#8] [Q&A]
Is it possible to get update version of tailwindcss? Currently, it is 3.0.24 but the devtools is using v3.0.13.

**Comment by shoaibsharif** [comment]
Right now tailwindcss version upgraded to 3.2 but I still didn't get hear anything.

**Comment by Aetles** [comment]
Any news on this? Devtools is still on 3.0.13 while Tailwind is on 3.3.7 now and 3.4 is close to release.
